cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -35,6 +35,33 @@ To read more about the execution plan operators mentioned in this chapter, see <
 [[how-are-queries-executed]]
 == How are queries executed?
 
+_***XXTODO by Docs Team - change this heading to something like 'Preparser options' (this is what they are called internally), or something more reader-friendly (if a term other than 'preparser' is decided upon, please also change accordingly in the text). This section no longer deals with 'how queries are executed' per se. More of that content will be added by me later on. END TODO***_
+
+[abstract]
+--
+This section describes the preparser options available in Cypher.
+--
+
+Queries are able to be fine-tuned through the use of _preparser options_.
+If there is a requirement to use one or more of these options, the query must be prepended with `CYPHER`, followed by the preparser option(s), as exemplified thus: `CYPHER preparser-option1 [further preparser options] query`.
```

Comment:
Perhaps simplified to 'In order to use these options, the query must be prepended with ...'

Reply:
👍 (slight amendment: wanted to retain and emphasise "one or more")